### ISSUE TITLE
Integrate name transfer service

### ISSUE THREAD

**Issue by larrysalibra** [blockstack/blockstack-browser#537]
@kantai has written some components of a transfer service which are currently located here:
https://github.com/kantai/blockstack-transfer-service

It includes code to migrate names from core wallet and portal pre-`v0.9.0`. Code for wallets after that and before`v0.10.0-alpha.1`is not yet written.

# Browser page

We'll need to put this together in the browser. The interface of the app should be pretty simple:

1. `textarea` to paste `wallet.json` or mnemonic to migrate or button to migrate current browser wallet. If current wallet, user will need to enter their password. 
2. User clicks continue
3. Service searches for names in the wallet and displays a list of names along with a box next to each for the address to which the name should be transferred. Each name has a check box which the user can uncheck if she does not want to transfer that name.
4. User clicks Transfer.

# Subsidizer service

We need a subsidizer service based on this test subzidizer with a hot wallet: https://github.com/kantai/blockstack-transfer-service/blob/master/python-service/test_subsidizer.py

@kantai will need code from Onename to finish implementing this.

**Comment by jackzampolin** [comment]
Is this issue still applicable?

**Comment by kantai** [comment]
Yes -- there's still people who will need help migrating names from older browser versions, and we still don't have a tool for it. I have a script that can output the requisite private keys, and then that can be sent over to a local blockstack API endpoint to do the transer/update, but we would need to flesh it out and document it before declaring victory there.

**Comment by jackzampolin** [comment]
Is this an REST call we could add to the `blockstack api`? Seems like code is in Python and could be added to a method in [`rpc.py`](https://github.com/blockstack/blockstack-core/blob/master/blockstack_client/rpc.py#L228). Is that the implementation path you were thinking?

**Comment by kantai** [comment]
No -- it's mostly javascript actually -- it takes the 24 word mnemonic and turns it into a private key hex string, which can then be passed to a REST call. This could be done by the browser, or it could be a tiny CLI script. I'd prefer not to add any more REST calls to `rpc.py`

**Comment by larrysalibra** [comment]
I've started work on a tool to let people calculate their private keys for older keychain formats here: https://github.com/blockstack/blockstack-browser/tree/feature/migration-tool
